### ISSUE TITLE
Add Laravel 10 support

### ISSUE THREAD

**Issue by pedzed** [GrahamCampbell/Laravel-Markdown#174]
This is currently not working for Laravel 10:
```sh
$ composer require graham-campbell/markdown:^14.0
```
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires graham-campbell/markdown ^14.0 -> satisfiable by graham-campbell/markdown[v14.0.0, 14.0.x-dev].
    - graham-campbell/markdown[v14.0.0, ..., 14.0.x-dev] require illuminate/contracts ^8.75 || ^9.0 -> found illuminate/contracts[v8.75.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev] but these were not loaded, likely because it conflicts with another require.


Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by stefnats** [comment]
I made a PR #175

**Comment by ultrono** [comment]
FYI, for anyone coming here looking for Laravel 10 support I've forked this repository. All tests are passing with Laravel 10. See https://github.com/GrahamCampbell/Laravel-Markdown/pull/175#issuecomment-1434999500.

Have been running this fork now for a few days now, on a system that gets a decent amount of use, without issue.

Hopefully #175 will get merged soon 🤖

**Comment by maxxscho** [comment]
Any chances #175 gets merge soon?

**Comment by cworreschk** [comment]
@GrahamCampbell Any news on #175 ?

**Comment by dsampaolo** [comment]
@GrahamCampbell Please merge this PR :)

**Comment by GrahamCampbell** [comment]
Thanks for making a start here, though there are quite a few missing things from that PR, such as updating the readme, and the tests are currently failing. I will handle the upgrade myself, shortly. In the meanwhile, [Sponsoring me](https://github.com/sponsors/GrahamCampbell) is a great way to support this project, and better enable me to perform these upgrades in a timely manor. I maintain or co-maintain upwards of 100 OSS repos. 💻

**Comment by GrahamCampbell** [comment]
>All tests are passing with Laravel 10. See https://github.com/GrahamCampbell/Laravel-Markdown/pull/175#issuecomment-1434999500.

Unfortunately, they are not. If you look closely, they are actually running on Laravel 9, not 10. This is because I have to first apply upgrades to my testbench packages.

**Comment by GrahamCampbell** [comment]
v15.0.0 has now been released, with PHP 8.2 and Laravel 10 support.

**Comment by ultrono** [comment]
> 

Ah, good catch, didn't notice that, can see the actions file uses "matchers", which I've not come across before. My fork has been running for well over a week without issue, with a simple composer change for the Laravel version.

Not that any of this matters now :)
